pkg/v2/mcapi.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2023 Seagate Technology LLC and/or its Affiliates
+
+package mcapi
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/Seagate/seagate-exos-x-api-go/pkg/client"
+	openapiclient "github.com/Seagate/seagate-exos-x-api-go/pkg/client"
+	"github.com/Seagate/seagate-exos-x-api-go/pkg/common"
+	"k8s.io/klog/v2"
+)
+
+// Client : Can be used to request the API
+type Client struct {
+	Ctx        context.Context
+	Username   string
+	Password   string
+	Addr       string
```

Comment:
Should this be an array instead, to support more than one controller?  Or defined as string containing a comma-separated list of addresses?

Reply:
I agree this needs to be addressed, but I need more time and discussion on what we should do.